### DISCUSSION TITLE
ERROR in ./node_modules/@aws-amplify/api-graphql/node_modules/graphql/language/printer.mjs 7:0-34

### DISCUSSION THREAD

**Discussion by ndilanwil** [webpack/webpack#17200] [General]
I am using the latest version of npm, graphql and react. but when starting my program I get a bunch of errors(all of them are as follows)

ERROR in ./node_modules/@aws-amplify/api-graphql/node_modules/graphql/language/printer.mjs 7:0-34
Module not found: Error: Can't resolve './visitor' in 'C:\Users\Dilan\Desktop\qwasar\my_yelp\node_modules@aws-amplify\api-graphql\node_modules\graphql\language'
Did you mean 'visitor.mjs'?

When checking the index.mjs file I see that all the files from which it is importing has the following tag
"Could not find a declaration file for module './printLocation.mjs'. 'c:/Users/Dilan/Desktop/qwasar/my_yelp/node_modules/graphql/language/printLocation.mjs' implicitly has an 'any' type."

**Comment by alexander-akait** [comment]
It means your package import something what doesn't exits or your resolver has invalid configuration
